iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/impl/bicep/TargetScopeDeclarationImpl.java

```diff
@@ -54,7 +54,8 @@ public Kind getKind() {
   public File.Scope scope() {
     if (expression.is(Kind.STRING_LITERAL)) {
       StringLiteral stringLiteral = (StringLiteral) expression;
-      switch (stringLiteral.value()) {
+      String stringLiteralValueUnquoted = stringLiteral.value().substring(1, stringLiteral.value().length() - 1);
```

Comment:
I'm not actually sure, whether quotation marks should be part of string literals value or should be stored as separate fields in the `StringLiteralImpl` class?

Reply:
I had the same concern in my PR. Let's discuss it offline.

Reply:
So we decided to have quotation marks as separate fields. This will help later to have simpler logic in Tree classes and in checks.